src/microbe_stage/MicrobeHUD.cs

```diff
@@ -249,10 +249,13 @@ protected override bool ShouldShowAgentsPanel()
         var colony = stage!.Player!.Colony;
         if (colony == null)
         {
-            return GetPlayerUsefulCompounds()!.IsSpecificallySetUseful(oxytoxy);
+            return GetPlayerUsefulCompounds()!.IsSpecificallySetUseful(oxytoxy) ||
+                GetPlayerUsefulCompounds()!.IsSpecificallySetUseful(mucilage);
```

Comment:
Not sure if `GetPlayerUsefulCompounds` runs a lot of computations or not, but in the former case it might be a good idea not to call it twice.